typescript-audit-logs-example/README.md

```diff
@@ -0,0 +1,74 @@
+# TypeScript Example App with Audit Logs powered by WorkOS
+
+An example Node.js / TypeScript application demonstrating how to use the [WorkOS Node.js SDK](https://github.com/workos/workos-node) to send and retrieve Audit Log events. This app is not meant to show a real-world example of an Audit Logs implementation, but rather to show concrete examples of how events can be sent using the Node.js SDK.
+
+## Prerequisites
+
+-Node.js version 10+
+
+## Node Project Setup
+
+1. Clone the main repo and install dependencies for the app you'd like to use:
+
+    ```bash
+    # HTTPS
+    git clone https://github.com/workos/typescript-example-applications.git
+    ```
+
+    or
+
+    ```bash
+    # SSH
+    git clone git@github.com:workos/typescript-example-applications.git
+    ```
+
+2. Navigate to Audit Logs app within the cloned repo.
+
+    ```bash
+    $ cd node-example-applications/typescript-audit-logs-example
```

Comment:
@logangingerich I believe the directory name is incorrect here? Should it be `typescript-example-applications`?

Reply:
ahhh good catches. Updating now 👀